rest.yaml

```
@@ -249,12 +249,927 @@ paths:
+      type: "object"
+      properties:
+        id: 
+          type: "string"
+          format: "uuid"
+        name: 
+          type: "string"
+        shortName: 
+          type: "string"
+        address: 
+          $ref: "#/components/schemas/Address"
+        website: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)